tests/unit_tests/blas/batch/axpy_batch_usm.cpp

```diff
@@ -46,15 +46,15 @@ namespace {
 template <typename fp>
 int test(device *dev, oneapi::mkl::layout layout, int64_t group_count) {
     // Catch asynchronous exceptions.
-    auto exception_handler = [](exception_list exceptions) {
+    cl::sycl::async_handler exception_handler = [](exception_list exceptions) {
```

Comment:
Why is it even necessary to not use `auto`? I think it should also work with `auto`.

Reply:
Mhmm, it doesn't seem to be necessary anymore... I recall that a few weeks ago there was an issue with hipSYCL regarding this.